### DISCUSSION TITLE
Feishin stopped showing a list of top tracks for artists.

### DISCUSSION THREAD

**Discussion by xusiai** [jeffvli/feishin#870] [Q&A]
For the past few months Feishin has not shown top tracks for artists. I rotated lastfm credentials but the issue persist. It fetches artist metadata, genres and a lastfm link, but no longer displays top tracks; even when lastfm has top track data available. I have no custom css and it happens across both Windows and Linux clients. I had a look around the discussion and issue boards but couldn't find an answer. Is this a known issue or intentional change?

**Comment by jeffvli** [comment]
![image](https://github.com/user-attachments/assets/479ae14d-7d8a-4a49-8233-91691f17c9b4)

Check that `Top songs` is enabled under the album artist page configuration.

**Comment by xusiai** [comment]
It is indeed enabled. I haven't touched the settings, but I double checked now and all 5 items are enabled.
